### DISCUSSION TITLE
TuYa IH012-RT01

### DISCUSSION THREAD

**Discussion by wokhen** [Koenkk/zigbee2mqtt#13331] [Devices]
Hi,
first thank you for your job.
I just purchased 2x TuYa IH012-RT01 human motion sensors.
After about 10-15 attempts I paired them with my home assistant using Sonoff Zigbee 3.0 USB Dongle Plus key and Z2M, but devices keep showing 'N/A' status for occupancy, battery_low and tamper values. I removed and paired again, no records in the registry.
Any tip is welcome, thank you
![immagine](https://user-images.githubusercontent.com/53829124/181879312-db1ec0c4-44d8-423f-bc09-4434db95f940.png)

**Comment by marine1988** [comment]
i have problems on reporting hell the state

**Comment by toxic0berliner** [comment]
I have the same device (IH012-RT01) and the same Sonoff Hub, running the latest hass and z2m (non-dev) and have the same issue of N/A value for this sensor.

**Comment by Koenkk** [comment]
If anyone has the orginal tuya gateway, please figure out the datapoints of this then we can probably fix this issue. https://www.zigbee2mqtt.io/advanced/support-new-devices/03_find_tuya_data_points.html#requirements-and-caveats

**Reply by Schlicki** [reply]
Do you still need? I have a original Tuya gateway I may use for setting it up.

**Comment by masakotsubuku1** [comment]
I had some of the same device (IH12-RT01) working fine with Conbee II for a while.  But since about a month, they are not available, emitting the error: 
MQTT publish: topic 'zigbee2mqtt/bridge/response/device/configure', payload '{"data":{"id":"Kitchen motion"},"error":"Device 'Kitchen motion' cannot be configured","status":"error","transaction":"5tm8h-1"}'

I added a SONOFF [SNZB-03] motion sensor at that time and then two Tuya IH12-RT01 sensors stop connecting. So I thought there might be conflicts between those 2 models. But maybe not. Anyway, until now, no solution found.

**Comment by vahaldor** [comment]
I just received three of these motions sensors, using Sonoff Zigbee 3.0 USB Dongle Plus key and Z2M and none of them could be recognized as supported device in HA.

**Comment by manjh** [comment]
I am going crazy trying to get this cheap and simple device to work.
Running Zigbee2MQTT on a Pi, with Conbee-2. 
Also running latest version of Domoticz with Zigbee2MQTT python plugin. 
Currently have five zigbee smoke detectors, and three Ikea light bulbs. Works fine.
Purchased one Zigbee PIR sensor, paired it. It comes up in the Zigbee2MQTT dashboard as IH012-RT01.
In Domoticz, the plugin creates four devices: for sensitivity, tamper, low battery and occupancy. 
I enabled the four devices with change.
So far, so good. 
But after this, whenever the device is triggered (I can see the blue LED light up), nothing happens in Domoticz! 
When changing the debug level in the plugin to "verbose", I can see a message coming in on the Domoticz log. 
But the devices are not updated.
Any idea what might be wrong?

**Reply by manjh** [reply]
Never mind. I decided it must be a bug in the Z2M Domoticz plugin, so I discarded the whole thing and moved to a newer AutoDiscover version. This now works fine.

**Comment by IPv777** [comment]
Hey, any news about this problem ?

**Comment by ffhjansen** [comment]
Mine worked, then stopped working (only reporting "occupied") and after removing and adding it again to z2m it's only stating 
    "keep_time": 120,
    "last_seen": "2023-06-12T14:36:12.212Z",
    "linkquality": 98,
    "sensitivity": "medium"
So no "occupancy" at all? Hopefully this is still an active issue?

**Reply by AndreiShx** [reply]
The same problems with TuYa IH012-RT01

**Comment by brio11** [comment]
Hey there,
I've recently bought TuYa IH012-RT01 which was about to run on HA with CC2531 zigbee2mqtt. Unfortunately it cannot be discovered by mqtt however it is stated to be supported:
[Zigbee2MQTT](https://www.zigbee2mqtt.io/devices/IH012-RT01.html)
Did anyone of you face the same issue and maybe solve it?

**Reply by metroeye1** [reply]
I have a same problem with you. I am getting mad. ;;

**Reply by brio11** [reply]
Do you also use CC2531? Which firmware version?

**Comment by Schlicki** [comment]
I have the issue that I can't change sensitivity. Always get 

`Publish 'set' 'sensitivity' to 'Bewegungsmelder Wohnzimmer' failed: 'Error: Write 0xa4c1389232a6b4a2/1 ssIasZone({"currentZoneSensitivityLevel":1}, {"timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":true,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (Timeout - 39327 - 1 - 113 - 1280 - 4 after 10000ms)'`
